File: www/casino/cgi/form-handler.php
<?php
header("Content-Type: text/html");
echo "<html><body><h1>PHP CGI</h1>";
if ($_SERVER['REQUEST_METHOD'] === 'POST') {
  echo "<p>POST received. Email: " . htmlspecialchars($_POST['email']) . "</p>";
} else {
  echo "<p>METHOD: " . $_SERVER['REQUEST_METHOD'] . "</p>";
  echo "<p>PATH_INFO: " . $_SERVER['PATH_INFO'] . "</p>";
}
echo "</body></html>";
?>

File: www/casino/cgi/form-handler.py
#!/usr/bin/env python3

import os
import cgi

print("Status: 200")
print("Content-Type: text/html")
print("")
print("<html><body>")
print("<h2>Hello from CGI</h2>")
form = cgi.FieldStorage()
if form.getvalue("name"):
    print(f"<p>Hi, {form.getvalue('name')}!</p>")
else:
    print("<p>No name provided</p>")

print("</body></html>")
